server/master/src/main/java/org/apache/accumulo/master/tableOps/CreateTable.java

```diff
@@ -58,6 +68,10 @@ public long isReady(long tid, Master environment) throws Exception {
     Utils.idLock.lock();
     try {
       tableInfo.tableId = Utils.getNextId(tableInfo.tableName, master.getInstance(), Table.ID::of);
+      if (tableInfo.props.containsKey(Property.TABLE_OFFLINE_OPTS + "create.initial.splits")
+          && this.splitFile != null) {
+        storeSplitFileNameInZooKeeper();
```

Comment:
I don't think this needs to be stored in ZK explicitly.  The way FATE works, each step of a FATE op is serialized to ZK.  Therefore if you store any info you want to keep around in the TableInfo object, then the TableInfo object will be serialized in subsequent steps.

Reply:
Updated TableInfo to contain a splitInfo entry. Enabled simplication of a couple of classes by negating the need for some utility methods that were needed when writing the info to ZK.